### DISCUSSION TITLE
Node ExperimentalWarning

### DISCUSSION THREAD

**Discussion by manicprogrammer** [decentralized-identity/veramo#1300] [Q&A] [answered]
Before I file an issue for this I wanted to get input from other folks. Any veramo command I run including `veramo -v` comes back with the following text before the output from the command.
` (node:7884) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time`

I am running :

- Veramo 5.5.3
- Node 20.10.0
- On Ubuntu 22.04.3 LTS from a WSL (Windows Subsystem for Linux) instance

Am I doing something wrong? Do I need to downgrade my node version? Has anyone else seen this? I'll file an issue if needed.

**Comment by mirceanis** [comment, marked as answer]
It's not your fault.
Current versions of veramo `import` some JSON-LD context files and that triggers the warning.

It's annoying, as `require`ing the same files doesn't show a warning.

I think it's a good idea to file an issue as it will help us track and refer to it until it's fixed.
